### DISCUSSION TITLE
Add phone number to "Call < username > buttons

### DISCUSSION THREAD

**Discussion by k8fish3r** [code4recovery/12-step-meeting-list#1463] [General]
Hi,

I used to amend buttons in TSML (OLD UI) via javascript.
I offset the changes running until after the page loaded to ensure the DOM was visible.
I don't seem to be able to edit the "meetings detail pages" in this way via javascript now.

Is there anything about the funky new TSML setup - operating as a single page application which would make the DOM elements uneditable via javascript now via document.getElementsByClassName ?

document.getElementsByClassName("css-1up26yg")[0].getElementsByTagName('a')[x].innerHTML +="<sub>Add subtext to button</sub>";

E.g. I'd like to edit the text on some meeting details buttons to TEXT , and adjust the action to send an SMS to the number via sms:// instead of call the number via tel: 
 
 Hopefully the picture below renders
<img width="889" alt="image" src="https://github.com/code4recovery/12-step-meeting-list/assets/67172278/af3669fa-d947-4e97-9b57-862ac2230081">

**Comment by joshreisner** [comment]
the blue phone button in the picture is for joining a meeting by phone, so that really needs to stay a call-only option

in this case we're really talking about this secondary contact button below

<img width="471" alt="Screenshot 2024-07-03 at 4 49 53 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/f697d2d4-7fa5-47ab-ad9f-71b3a50c974c">

we could potentially add a new field here, but what if we were just to change that globally to say `Text {name}` and use the `sms:` prefix everywhere? i feel like this is probably the preferred method 99.999999% of the time

**Reply by k8fish3r** [reply]
If its possible to offer SMS as a means of communication that would be ideal. If that's doable. I'm assuming a few meeting contacts prefer responding to SMS messages to calls whilst at work. It's alot more private method of sharing experience, strength and hope to prospective members.

**Reply by k8fish3r** [reply]
Personally I just don't add my mobile number (and add my email address instead).
